packages/ketcher-polymer-editor-react/src/styles/theme.js

```diff
@@ -52,7 +53,8 @@ const color = {
       primary: '#FFFFFF',
       secondary: '#005662',
       disabled: '#7A7A7A'
-    }
+    },
+    reset: '#FF2727'
```

Comment:
'reset' seems a bit too specific for a theme. We'll probably have just one button like this, i think we'd better style it on its own.